src/index.js

```diff
@@ -1 +1,35 @@
 import './style.css';
+/*eslint-disable*/ 
```

Comment:
Well done overall 💯 I read your PR description and based on the code you provided, the circular dependency might be caused by the fact that `./modules/get-post-data.js` is importing from `../index.js`, which in turn may import from `./modules/get-post-data.js`. Since everything else works as intended, I wanted to help you fix it. 👇🏽 

To fix this, you can refactor the code to remove the circular dependency. ♻️  One possible solution is to move the `display` and `refreshDom` functions to a separate module, which can be imported into both `./modules/get-post-data.js` and `../index.js`. This will allow you to remove the circular dependency and ensure that the shared code is only loaded once. 1️⃣ 

Here's an example of how you could refactor your code 👇🏽 

In a new module `./modules/display.js`:

```
const ul = document.querySelector('ul');

export const refreshDom = () => {
  ul.innerHTML = '';
};

export const display = (user, score) => {
  const li = `<li>${user}: ${score}</li>`;
  ul.innerHTML += li;
};
```

In the `./modules/get-post-data.js` module:
```
import { display, refreshDom } from './display.js';

export const api = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/3Y74LzLKCnfghIcTzPAQ/scores/';

export const getJSONData = async () => {
  refreshDom();
  const response = await fetch(api);
  const jsonData = await response.json();

  jsonData.result.forEach((item) => {
    display(item.user, item.score);
  });
};

export const postData = async (url = '', data = {}) => {
  const response = await fetch(url, {
    method: 'POST',
    headers: {
      'Content-Type': 'application/json',
    },
    body: JSON.stringify(data),
  });
  return response.json();
};
```

And in the `../index.js` module:

```
import './style.css';
import { api, getJSONData, postData } from './modules/get-post-data.js';
import { display } from './modules/display.js';

const user = document.querySelector('#user');
const score = document.querySelector('#score');
const form = document.querySelector('form');

const refreshBtn = document.querySelector('#refresh');

refreshBtn.addEventListener('click', getJSONData);

form.addEventListener('submit', (e) => {
  e.preventDefault();

  if ((user.value.length > 0) && (score.value.length > 0)) {
    postData(api, { user: user.value, score: score.value }).then(() => {
      display(user.value, score.value);
      user.value = '';
      score.value = '';
    });
  }
});

getJSONData();
```
By doing this, you should be able to eliminate the circular dependency ❌  and ensure that your code works as intended. 🤓